src/resources/css/reset.css

```diff
@@ -0,0 +1,370 @@
+/**
+ * This CSS file was auto-generated via PostCSS
+ *
+ * Contributors should avoid editing this file, but instead edit the associated
+ * src/resources/postcss/ file. For more information, check out our engineering
+ * docs on how we handle CSS in our engineering docs.
+ *
+ * @see: http://moderntribe.github.io/products-engineering/css/
+ */
+
+/* -----------------------------------------------------------------------------
+	*
+	* Global "Resets"
+	*
+	* ----------------------------------------------------------------------------- */
+
+	.tribe-events * {
+		-webkit-box-sizing: border-box;
+		        box-sizing: border-box;
+	}
+
+	/* -----------------------------------------------------------------------------
+	*
+	* HTML5 Elements
+	*
+	* ----------------------------------------------------------------------------- */
+
+	.tribe-events article,
+	.tribe-events aside,
+	.tribe-events details,
+	.tribe-events figcaption,
+	.tribe-events figure,
+	.tribe-events footer,
+	.tribe-events header,
+	.tribe-events main,
+	.tribe-events menu,
+	.tribe-events nav,
+	.tribe-events section,
+	.tribe-events summary {
+		display: block;
+	}
+
+	/* -----------------------------------------------------------------------------
+	*
+	* Normalize "Light"
+	*
+	* See normalize source code for clarity on rules
+	* https://github.com/necolas/normalize.css
+	*
+	* ----------------------------------------------------------------------------- */
+
+	.tribe-events html {
```

Comment:
How does this reset and the body one work. For the body one to work tribe-events class would have to be on the html el, but for this one, i dont see how it would work?